### ISSUE TITLE
Manifest text in certain manifests not compatible

### ISSUE THREAD

**Issue by swzCuroverse** [curoverse/tapestry#4]
Manifest text is not compatible in these collection manifests so I can't access them.  Ward ask me to post the ticket and he would rephrase it if necessary :)

Collections listed below:
Tapestry--my.pgp-hms.org--UserFile--790
Tapestry--my.pgp-hms.org--UserFile--783
Tapestry--my.pgp-hms.org--UserFile--594

**Comment by cure** [comment]
The manifest encoding was invalid for the first two collections; spaces were not escaped. This must have happened when they were created, a long time ago (buggy version of an Arvados SDK?).

I fixed those. The third one looks fine to me, I can access it.

**Comment by cure** [comment]
Ah, the third broken one is

Tapestry--my.pgp-hms.org--UserFile--594

I fixed that one too.
